package/package.json

```diff
@@ -20,5 +21,20 @@
     },
     "homepage": "https://github.com/Henrik-3/unofficial-valorant-api",
     "author": "HenrikDev",
-    "license": "ISC"
+    "license": "ISC",
+    "type": "module",
+    "devDependencies": {
+        "@types/node": "^18.7.5",
+        "@typescript-eslint/eslint-plugin": "^5.33.1",
+        "@typescript-eslint/parser": "^5.33.1",
+        "eslint": "^8.22.0",
+        "typedoc": "^0.23.10",
+        "typedoc-plugin-markdown": "^3.13.4",
+        "typescript": "^4.7.4"
+    },
+    "scripts": {
+        "docGen": "typedoc --plugin typedoc-plugin-markdown && node ./test/editDocs.js",
+        "tsc": "tsc",
+        "testTsc": "cd test && tsc"
```

Comment:
```suggestion
        "build:docs": "typedoc --plugin typedoc-plugin-markdown && node ./test/editDocs.js",
        "build:code": "tsc",
        "build:tests": "cd test && tsc"
```

Reply:
```json
"build:docs": "typedoc --plugin typedoc-plugin-markdown && node ./test/editDocs.js",
"build:tsc": "tsc --watch false",
"develop:tsc": "tsc"
```Ill change it to this, also Ill remove `testTsc` as I'll just remove the tests